SwiftUI_DZ/SwiftUI_DZ/Helpers/ShakeEffect.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ShakeEffect.swift
+//  SwiftUI_DZ
+//
+//  Created by Олег Яковлев on 29.01.23.
+//
+
```

Comment:
```suggestion
//
//  ShakeEffect.swift
//  SwiftUI_DZ
//
//  Created by Олег Яковлев on 29.01.23.
//

```
ай я яй кто то форматер отключил)